tilequeue/command.py

```diff
@@ -1979,7 +1979,8 @@ def tilequeue_batch_enqueue(cfg, args):
     logger = make_logger(cfg, 'batch_enqueue')
 
     import boto3
-    client = boto3.client('batch', region_name='us-east-1')
+    region_name = os.environ.get('AWS_DEFAULT_REGION', 'us-east-1')
+    client = boto3.client('batch', region_name=region_name)
```

Comment:
I would assume that boto3 would automatically check the environment variable for us, but this certainly doesn't hurt.

Reply:
I think so too. This was my attempt to keep the default the same as before, as I think that not setting the environment variable would otherwise be an error?

Reply:
Ah yea, good point. It does error out if there is no default region set.

👍